packages/manager/modules/support/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@ovh-ux/manager-support",
+  "version": "0.0.0",
+  "license": "BSD-3-Clause",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/umd/support.js",
+  "scripts": {
+    "build": "rollup -c --environment BUILD:production",
+    "dev": "rollup -c --environment BUILD:development",
+    "dev:watch": "yarn run dev --watch",
+    "prepare": "yarn run build",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-support' --include-filtered-dependencies -- yarn run build",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-support' --include-filtered-dependencies -- yarn run dev",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-support' --include-filtered-dependencies -- yarn run dev:watch"
+  },
+  "dependencies": {
+    "bootstrap": "^3.4.1",
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
```

Comment:
Maybe we should only use `bootstrap4` (and the latest version)

Reply:
`"bootstrap": "^3.4.1",` has been removed ;)